packages/core/src/testInteg/codecatalyst/devEnv.test.ts

```diff
@@ -52,40 +52,49 @@ describe('InactivityMessages', function () {
+        await devEnvActivity.sendActivityUpdate()
+        await devEnvActivity.sendActivityUpdate()
+        await devEnvActivity.sendActivityUpdate()
+        await devEnvActivity.sendActivityUpdate()
 
         await assertMessagesShown([
-            ['Your CodeCatalyst Dev Environment has been inactive for 10 minutes, shutting it down in 5 minutes.', 10],
-            ['Your CodeCatalyst Dev Environment has been inactive for 11 minutes, shutting it down in 4 minutes.', 11],
-            ['Your CodeCatalyst Dev Environment has been inactive for 12 minutes, shutting it down in 3 minutes.', 12],
-            ['Your CodeCatalyst Dev Environment has been inactive for 13 minutes, shutting it down in 2 minutes.', 13],
+            ['Your CodeCatalyst Dev Environment has been inactive for 10 minutes, and will stop in 5 minutes.', 10],
+            ['Your CodeCatalyst Dev Environment has been inactive for 11 minutes, and will stop in 4 minutes.', 11],
+            ['Your CodeCatalyst Dev Environment has been inactive for 12 minutes, and will stop in 3 minutes.', 12],
+            ['Your CodeCatalyst Dev Environment has been inactive for 13 minutes, and will stop in 2 minutes.', 13],
             ['Your CodeCatalyst Dev Environment has been inactive for 14 minutes, and will stop soon.', 14],
         ])
     })
 
     it('shows expected messages 5 minutes before shutdown on a 60 minute inactivity timeout', async function () {
-        await instance.setupMessage(60, devEnvActivity as unknown as DevEnvActivity, relativeMinuteMillis)
+        await inactivityMsg.init(60, devEnvActivity as unknown as DevEnvActivity, relativeMinuteMillis)
+        setInitialOffset(57)
+        await devEnvActivity.sendActivityUpdate()
+        await waitUntil(async () => actualMessages.length > 0, { interval: 10 })
+        setInitialOffset(58)
+        await devEnvActivity.sendActivityUpdate()
+        await waitUntil(async () => actualMessages.length > 2, { interval: 10 })
 
         await assertMessagesShown([
-            ['Your CodeCatalyst Dev Environment has been inactive for 55 minutes, shutting it down in 5 minutes.', 55],
-            ['Your CodeCatalyst Dev Environment has been inactive for 56 minutes, shutting it down in 4 minutes.', 56],
-            ['Your CodeCatalyst Dev Environment has been inactive for 57 minutes, shutting it down in 3 minutes.', 57],
-            ['Your CodeCatalyst Dev Environment has been inactive for 58 minutes, shutting it down in 2 minutes.', 58],
-            ['Your CodeCatalyst Dev Environment has been inactive for 59 minutes, and will stop soon.', 59],
+            ['Your CodeCatalyst Dev Environment has been inactive for 57 minutes, and will stop in 3 minutes.', 0],
+            ['Your CodeCatalyst Dev Environment has been inactive for 58 minutes, and will stop in 2 minutes.', 0],
+            ['Your CodeCatalyst Dev Environment has been inactive for 59 minutes, and will stop soon.', 1],
```

Comment:
This test is not working perfectly. But we should probably remove the progress message anyway. There is no real reason to show multiple messages, because if the user isn't around to see it, it won't matter how many messages are shown.